content/en/security/guide/security-findings-migration.md

```diff
@@ -42,31 +43,30 @@ If you need assistance with your migration, contact [Datadog support][14].
 
 ### New features
 
-Starting January 2026, the following features use the new data model:
+Starting late January 2026, the following features use the new data model:
 - Unified search queries for Cloud Security [Misconfigurations][9], [Identity Risks][10], [Attack Paths][11], [IaC][12], and [API][13] findings
 - A new unified public findings API
 - [Dashboard][2] support for Code Security
 - Graphing security findings in [Datadog Sheets][3]
 - Datadog [Workflow Automation][4] support for all finding types
 - Using SQL to query security findings and join them with other Datadog telemetry using [DDSQL Editor][5]
 
-With additional future changes:
+The following additional features will be released later:
 
-- Cloud Security Vulnerabilities
-- Code Security findings (SCA, SAST, IAST, Secrets)
+- Unified search queries for Cloud Security Vulnerabilities and Code Security findings (SCA, SAST, IAST, Secrets)
```

Comment:
Technically unified with the above right? 
Maybe need to say something like "Cloud Security Vulnerabilities and Code Security findings (SCA, SAST, IAST, Secrets) become part of the unified search API"

Otherwise it sounds like these two specifically are getting merged or something